__tests__/app.test.js

```diff
@@ -119,10 +117,21 @@ describe("app", () => {
 
 
   describe("/api/articles/:article_id/comments", () => {
+    test('Returns Error with 404 when the article id it does not exist', () => {
+      const article_id = 50;
+      return request(app)
+     .get(`/api/articles/${article_id}/commentS`)
+      .expect(404)
+      .then(({body})=>{
+      expect(body.msg).toBe("Article not found")
+
+        });
+      
+    });
```

Comment:
I would recommend doing the happy path first then error handling after.

There is also one more error handling test that can be added for this endpoint, that would be if the article id is invalid i.e. a word instead of a number